clickhouse/.cjs.swcrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
as you are integrating this plugin into our multi plugin repo, this file is not needed